packages/web-frontend/src/containers/OverlayDiv/components/NotFoundPage.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+import { PrimaryButton } from '../../../components/Buttons';
+import { goHome } from '../../../actions';
+import { connect } from 'react-redux';
+
+const Container = styled.div`
+  height: 500px;
+  width: 800px;
+  max-width: 100%;
+  max-height: 100%;
+  text-align: center;
+  padding-top: 20%;
+  padding-bottom: 30px;
+`;
+
+const Image = styled.img`
+  width: 300px;
+  max-width: 100%;
+  margin-bottom: 30px;
+`;
+
+const Text = styled(Typography)`
+  font-size: 18px;
+  line-height: 21px;
+  color: white;
+  margin-bottom: 30px;
+`;
+
+const StyledButton = styled(PrimaryButton)`
+  width: 300px;
+`;
+
+const NotFoundPageComponent = ({ goHome }) => {
+  return (
+    <Container>
+      <Image src="/images/404.svg" />
```

Comment:
Strictly speaking, having text in images isn't recommended, mainly because any user using assistive technologies won't be able to interpret it. There are a couple of options here:

- Instead of using an image, make this into a heading (probably h2) and style to look like the image
- If you need it to be an image, adding alt text to the image to describe what it is will help the screenreader read out what it says. 

My inclination would be to style a heading to look like the image, purely because the image itself is just text, so there is nothing else to display except '404'.

Reply:
Yes good point. I think that alt text should achieve the same goal for a screen reader right (and I should have added that in the first place obviously) without having to change the design. I don't think that I can make it look exactly like the image with just css as I think it's a different font. Plus using alt text I can put a more human centric text such as page not found.